packages/chip/src/Chip/Chip.styles.ts

```diff
@@ -39,156 +47,249 @@ export const chipWrapperPadding: Record<
+  [Theme.Light]: palette.gray.light1,
 };
 
-/**
- * Chip wrapper
- */
-export const chipWrapperThemeStyle = (variant: Variant, theme: Theme) => css`
-  color: ${variantColor[variant][theme].color};
-  background-color: ${variantColor[variant][theme].bgColor};
-  transition: background-color ${transitionDuration.faster}ms ease-in-out;
-`;
+const wrapperDisabledColor = {
+  [Theme.Dark]: palette.gray.dark2,
+  [Theme.Light]: palette.gray.base,
+};
 
-export const chipWrapperBaseStyle = css`
+export const wrapperBaseStyles = (
+  baseFontSize: BaseFontSize,
+  variant: Variant,
+  theme: Theme,
+) => css`
   display: inline-flex;
   align-items: center;
   overflow: hidden;
   white-space: nowrap;
-  box-sizing: border-box;
   border-radius: 4px;
-`;
-
-export const chipWrapperSizeStyle = (baseFontSize: BaseFontSize) => css`
   font-size: ${fontSize[baseFontSize]}px;
   line-height: ${lineHeight[baseFontSize]}px;
-`;
 
-export const chipWrapperBaseDisabledStyles = css`
-  cursor: not-allowed;
+  color: ${variantColor[variant][theme].text.default};
+  background-color: ${variantColor[variant][theme].background.default};
+  transition: background-color ${transitionDuration.faster}ms ease-in-out;
 `;
 
-export const chipWrapperDisabledStyle: Record<Theme, string> = {
-  [Theme.Light]: css`
-    color: ${palette.gray.base};
-    background-color: ${palette.gray.light3};
-  `,
-  [Theme.Dark]: css`
-    color: ${palette.gray.dark2};
-    background-color: ${palette.gray.dark4};
-    box-shadow: inset 0 0 1px 1px ${palette.gray.dark2};
-  `,
-};
+export const wrapperDisabledStyles = (theme: Theme) =>
+  cx(
+    css`
+      cursor: not-allowed;
+      background-color: ${color[theme].background.secondary.default};
+      color: ${wrapperDisabledColor[theme]};
+    `,
+    {
+      [css`
+        box-shadow: inset 0 0 1px 1px ${palette.gray.dark2};
+      `]: theme === Theme.Dark,
+    },
+  );
 
 /**
- * Chip text
+ * Chip wrapper
  */
-export const chipTextSizeStyle = (baseFontSize: BaseFontSize) => css`
+export const getWrapperStyles = (
+  baseFontSize: BaseFontSize,
+  variant: Variant,
+  theme: Theme,
+  isDisabled = false,
+) =>
+  cx(wrapperBaseStyles(baseFontSize, variant, theme), {
+    [wrapperDisabledStyles(theme)]: isDisabled,
+  });
+
+export const textBaseStyles = (
+  baseFontSize: BaseFontSize,
+  variant: Variant,
+  theme: Theme,
+) => css`
   padding-inline: ${chipWrapperPadding[baseFontSize].x}px;
   padding-block: ${chipWrapperPadding[baseFontSize].y}px;
-`;
 
-export const chipTextDismissSizeStyle = css`
-  padding-inline-end: 2px;
-`;
+  display: flex;
+  gap: 2px;
+
+  svg {
+    align-self: center;
+    color: ${variantColor[variant][theme].icon.default};
+  }
 
-export const chipTextStyles = (variant: Variant, theme: Theme) => css`
   &:focus-within {
-    background-color: ${variantColor[variant][theme].focusBgColor};
+    background-color: ${variantColor[variant][theme].background.focus};
   }
```

Comment:
Does this selector ever get used? The text span wrapping the glyph/text doesn't seem to ever have `:focus-within` unless it's forcibly done through dev tools
![Screenshot 2024-07-18 at 2 47 57 PM](https://github.com/user-attachments/assets/21d60708-975c-4819-a338-67c9fca1332b)
I'm seeing that there is a `label` that can get passed in. Is this selector included here if a consumer passes in a focusable label?

Reply:
This is used when the chip truncates.

Reply:
oh gotcha, I see now